packages/node_modules/nav-frontend-alertstriper/src/alertstripe.tsx

```diff
@@ -58,15 +62,14 @@ export interface AlertStripeBaseProps extends AlertStripeProps {
 
 class AlertStripe extends React.Component<AlertStripeBaseProps> {
     render() {
-        const { type, solid, size, className, children, ...props } = this.props;
-        const ariaLive = type === 'advarsel' || type === 'feil' ? 'assertive' : undefined;
+        const { type, solid, size, className, children, role } = this.props;
 
         return (
-            <div className={cls(type, solid, className)} {...props} aria-live={ariaLive}>
+            <div className={cls(type, solid, className)} role={role} >
```

Comment:
Bør kanskje beholde `{...props}`-spread her?

Reply:
Det er ikke andre props som kan sendes inn, så da blir det vel enklere å sette role direkte? Enklere å lese at dette er alt som settes.

Reply:
Hm, liker som regel å spre øvrige props på rot-elementet som default. Vanskelig å forutsi hvilke HTML-attributter man kan få bruk for. F.eks. kan det tenkes noen vil putte på en `aria-label` for å gjøre det enda tydeligere for skjermleser-brukere at elementet er faktisk er en "feilmelding" eller en "infoboks" eller noe sånt. Innebygget fleksibilitet 🙂

Reply:
Samtidig som det åpner for mye rart - tenker ofte det er bedre å da tilby aria-label som en prop dersom det blir behov :D - men jeg skal fikse